lib/message_builder.rb

```diff
@@ -71,7 +71,7 @@ def pull_requests
   end
 
   def dependapanda
-    @dependapanda ||= pull_requests.select { |pr| pr[:author].include?("dependabot") }
```

Comment:
Have we come across any PRs where `author` isn't specified? Defensive code is good but it's added complexity if not needed.

EDIT: looks like you remove this code in the next commit anyway, so I'm not sure why we're bothering changing it here? Instinctively feels like we're missing an accompanying test here too.